src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -203,6 +203,18 @@ internal static partial class ControlDevel
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Bridge_Remove_Popup")]
             public static extern void DaliAccessibilityBridgeRemovePopup(global::System.Runtime.InteropServices.HandleRef arg1);
 
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Accessible_GetCurrentlyHighlightedActor")]
+            public static extern global::System.IntPtr DaliAccessibilityAccessibleGetCurrentlyHighlightedActor();
+
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Accessible_GetHighlightActor")]
```

Comment:
@grzelewski 
Hello
now I put the \*\***NO MERGE**\*\* tag on thie PR.
you can remove \*\***NO MERGE**\*\* tag for yourself when this PR is fixed and ready to be merged.
thank you.

Reply:
Done.